main.py

```diff
@@ -1009,9 +1009,6 @@ def get_redirect_route(
     get_redirect_route(
         r'%s' % feconf.BLOG_DASHBOARD_DATA_URL,
         blog_dashboard.BlogDashboardDataHandler),
-    get_redirect_route(
-        r'%s' % feconf.BLOG_DASHBOARD_URL, blog_dashboard.BlogDashboardPage),
```

Comment:
Can we remove `BLOG_DASHBOARD_URL` from feconf? (Only if it's not used elsewhere)